app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class FoodsController < ApplicationController
+  before_action :set_food, only: %i[show edit update destroy]
+
+  # GET /foods or /foods.json
+  def index
+    @foods = Food.all
```

Comment:
- [ ] As the previous reviewer said, please render the foods belonging to the current user. You can use the 'current_user' method that comes with the devise.

Reply:
well resolved! ✔️